dev/RadioMenuFlyoutItem/RadioMenuFlyoutItem.cpp

```diff
@@ -50,31 +67,51 @@ void RadioMenuFlyoutItem::OnInternalIsCheckedChanged(const winrt::DependencyObje
     else if (!IsChecked())
     {
         IsChecked(true);
-        UpdateSiblings();
+        UncheckPreviousIfNecessary();
     }
 }
 
-void RadioMenuFlyoutItem::UpdateSiblings()
+void RadioMenuFlyoutItem::UncheckPreviousIfNecessary()
```

Comment:
nit: This method also selects this item as the selected item for this group, so the naming could be SetSelectedItemForGroupAndUnselectPrevious to make it super clear.

Reply:
Good point. I renamed it.